### PR TITLE
Updating hand documentation

### PR DESCRIPTION
**Pull request by georgiablanco**
## Proposed changes

Updating hand documentation from feedback

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [x] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [x] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

**Review by jeferal — APPROVED** [review]

File: docs/index.rst [hunk 1]
```diff
@@ -175,7 +175,7 @@ Maintaining the system
 
 * :doc:`/user_guide/mh_mechanical_maintenance`
 * :doc:`/user_guide/mh_electronic_maintenance`
-* :doc:`/user_guide/mh_installing_the_software`
+* :doc:`/user_guide/mh_re_installing_the_software`
```
Comment (beatrizleon) [inline docs/index.rst:178]: I dont think you need to change that here, as you haven't renamed the file. Inside it you can change the title as you have done.

File: docs/user_guide/fc_FAQ.rst [hunk 2]
```diff
@@ -42,10 +42,16 @@ Software
...
+A: Check that the Hand is powered on and that the Ethernet cable is connected.
+Go to the "Shadow Advanced Launchers" folder on the desktop and click on the "Launch NUC Container" icon. This will open a terminal window. Type the following command:
+
+
+.. code-block:: shell
+
+    sr_hand_detector_node
+
+
+If the Hand is connected, you should see a message saying that a Hand is connected and the ethernet adapter it is connected to. If the Hand is not connected, you will see a message saying that no Hand is detected. If this is the case, unplug and replyg the Ethernet cable and Power cycle the hand and try again.
```
Comment (beatrizleon) [inline docs/user_guide/fc_FAQ.rst]:
```suggestion
If the Hand is connected, you should see a message saying that a Hand is connected and the ethernet adapter is connected to. If the Hand is not connected, you will see a message saying that no Hand is detected. If this is the case, unplug and replug the Ethernet cable and Power cycle the hand and try again.
```

File: docs/user_guide/sd_command_line.rst [hunk 3]
```diff
@@ -300,12 +355,108 @@ Trajectory Controller
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+- Command
+
+  .. code-block:: shell
+  
+     /rh_wr_trajectory_controller/command
+
+  This topic can be published to and is the set position for the wwrist trajectory controller. It comprises an array of all the joints set positions and is used for commanding the robot. For example the rqt joint sliders publish to it.
```
Comment (beatrizleon) [inline docs/user_guide/sd_command_line.rst]:
```suggestion
  This topic can be published to and is the set position for the wrist trajectory controller. It comprises an array of all the joints set positions and is used for commanding the robot. For example the rqt joint sliders publish to it.
```

**Review by beatrizleon — CHANGES_REQUESTED** [review]

File: docs/index.rst (hunk 1 above)
Comment (beatrizleon) [inline docs/index.rst:178]: you also need to change this in line 187

**Comment by beatrizleon** [comment]
I added a few comments. Also we need to solve the conflicts.

**Review by beatrizleon — APPROVED** [review]

**Pull request merged**